lib/pytest-lsp/tests/examples/server-stderr/server.py

```diff
@@ -0,0 +1,21 @@
+import sys
+
+from lsprotocol import types
+from pygls.server import LanguageServer
+
+server = LanguageServer("server-stderr", "v1")
+
+
+@server.feature(types.TEXT_DOCUMENT_COMPLETION)
+def completion(params: types.CompletionParams):
+    items = []
+
+    for i in range(10):
+        print(f"Suggesting item {i}", file=sys.stderr, flush=True)
```

Comment:
TIL `stderr` was only changed to be line-buffered by default in Python 3.9, so to work in 3.8 we need to set `flush=True` here